src/main/java/org/folio/circulation/domain/NoteTypesRepository.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.circulation.domain;
```

Comment:
Recently, the repositories were all moved to a different package. Please move this to `org.folio.circulation.infrastructure.storage.notes`